.travis.yml

```diff
@@ -1,19 +1,26 @@
 language: go
 
+branches:
+  except:
+    - master
+  only:
+    - release
 go:
   - 1.13.x
```

Comment:
should we update to 1.14.x version, which is the latest.

Reply:
@mallow111 not sure if the source is tested with  golang 1.14.x, hence just use 1.13.x